src/utils.ts

```diff
@@ -300,3 +300,55 @@ export const compactNumber = new Intl.NumberFormat('en', {
   //@ts-ignore
   compactDisplay: 'short',
 }).format;
+
+export const getMonthsByCycle = (
+  billingCycle: 'monthly' | 'quarterly' | 'half-yearly' | 'yearly',
+) => {
+  switch (billingCycle) {
+    case 'monthly':
+      return 1;
+    case 'quarterly':
+      return 3;
+    case 'half-yearly':
+      return 6;
+    case 'yearly':
+      return 12;
+    default:
+      return 1;
+  }
+};
+
+interface IPlan {
+  type: string;
+  credits: number;
+  subscriberLimit: number;
+  emailsByMonth: number;
+}
+
+export const getPlanDescription = (plan: IPlan) => {
```

Comment:
I can move this to `plan-service.ts`, what do you think?

Reply:
Seems more natural, and perhaps change the name of the text, move it to general instead of being a part of plan_calculator texts.

Reply:
I got confused with the other function, my mistake 😅 
Beyond that, this could be useful for:

![image](https://user-images.githubusercontent.com/10213170/123811529-b4e71000-d8c9-11eb-89a0-1968769032e3.png)

Reply:
Yes, this function returns an object with the amount in each case (that it could be credits, subscribers, etc) and and the description ID that is the one you have to use it to find the correctly content in language file.